system/datatypes/mailinglist.php

```diff
@@ -125,6 +125,11 @@ public function process(HyphaRequest $request) {
 		return '404';
 	}
 
+	public function getSortDateTime() {
+		// TODO: Return last mailing sent date?
```

Comment:
Is this WIP or left for a later time/issue?

Reply:
Left for later indeed.